whiskey.yml

```diff
@@ -34,50 +36,56 @@ Build:
     OnlyDuring: Clean
     Path: reset.ps1
 
-- Pester4:
-    Script: Tests\*.Tests.ps1
+- Pester:

```

Comment:
Move this to be the last task. It should test the module as its going to be published.